routes/messages/index.js

```diff
@@ -0,0 +1,54 @@
+const express = require('express');
+
+const router = express.Router();
+
+module.exports = (middlewares) => {
+  const { db } = middlewares;
+
+  router.get('/:id', async (req, res) => {
+    const id = req.query;
+
+    try {
+      const messages = await db.messages.listMessage(id);
+      res.status(200).json(messages);
+    } catch (error) {
+      res.send(500, error.message);
+    }
+  });
+
+  router.post('/:id', async (req, res) => {
```

Comment:
I am not sure what your problem you were describing it here? Do you have to use /:id when testing? I checked out the project and had it say just `router.post('/', async (req, res) => {` and it worked.

But I noticed that your error handling is not working correctly! If I post nonsense data, I get back a 201 status and the "insertedMessage" will be an error from knex.

Reply:
Yes, it is just a problem I'm struggled with. If there is '/:id', I can get the data and insert data into the database. If I delete it, I cannot get the data.